tech-docs/README.md

```diff
@@ -0,0 +1,70 @@
+# HMPPS Integration API Documentation Site
+
+[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=flat&logo=github&label=MoJ%20Compliant&query=%24.result&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fapi%2Fv1%2Fcompliant_public_repositories%2Ftemplate-documentation-site)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/public-github-repositories.html#template-documentation-site "Link to report")
+
+## Contents
+
+- [About this project](#about-this-project)
+- [Getting started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+  - [Running the application](#running-the-application)
+  - [Running checks](#running-checks)
+  - [Updating configuration](#updating-configuration)
+- [Publishing](#publishing)
+
+## About this project
+
+This repository is used to generate the [technical documentation website](https://ministryofjustice.github.io/hmpps-integration-api-docs) for the [HMPPS Integration API](https://github.com/ministryofjustice/hmpps-integration-api).
```

Comment:
Do the references to the docs repo need to be updated in here?

Reply:
Thanks - yes updated this and the Makefile